opendevin/core/metrics.py

```diff
@@ -0,0 +1,54 @@
+class Metrics:
+    """
+    Metrics class can record various metrics during running and evaluation.
+    Currently we define the following metrics:
+        total_cost: the total cost of the current LLM.
+    """
+
+    def __init__(self) -> None:
+        self._total_cost: float = 0.0
+        self._costs: list[float] = []
+        self.reset()
```

Comment:
this could be removed?

Reply:
Keep it or lint will report some problem.